roles/servicetelemetry/templates/manifest_grafana_ds.j2

```diff
@@ -1,7 +1,7 @@
 apiVersion: integreatly.org/v1alpha1
 kind: GrafanaDataSource
 metadata:
-  name: {{ meta.name }}-ds-prometheus
+  name: {{ meta.name }}-datasources
```

Comment:
Will the dashboards will need to be adjusted for this? This is just a local naming convention in Grafana right?

Reply:
Never mind I looked through the code. This looks good.

Reply:
this won't affect the dashboards at all. This change is purely to make the documentation less confusing